odoo/modules/db.py

```diff
@@ -62,9 +62,10 @@ def initialize(cr):
             info['application'], info['icon'],
             info['sequence'], info['summary']))
         id = cr.fetchone()[0]
-        cr.execute('INSERT INTO ir_model_data \
-            (name,model,module, res_id, noupdate) VALUES (%s,%s,%s,%s,%s)', (
-                'module_'+i, 'ir.module.module', 'base', id, True))
+        cr.execute("""
+            INSERT INTO ir_model_data (name,model,module, res_id, noupdate, create_date, write_date)
+            VALUES (%s,'ir.module.module','base',%s,true,now() at time zone 'UTC', now() at time zone 'UTC')
+            """, ('module_'+i, id))
```

Comment:
I am in favor of using SQL defaults for `create_date` and `write_date`, as this enables to simplify code in several places, without introducing too much magic.
But what about `create_uid` and `write_uid`?  Those ones are apparently *never* assigned.

Reply:
> But what about create_uid and write_uid? Those ones are apparently never assigned.

Correct. Didn't really want to touch that part, as they've never been written to these model and there was nothing similar (the weird fields I'm removing only cover the date part of log_access)